pype/plugins/global/publish/integrate_new.py

```diff
@@ -90,7 +90,9 @@ class IntegrateAssetNew(pyblish.api.InstancePlugin):
                 "harmony.template",
                 "harmony.palette",
                 "editorial",
-                "background"
+                "background",
+                "renderLayer",
```

Comment:
we don't need to add `renderLayer` and `renderPass`. the frames must be reclasified as `render` by the time they get here.